README.md

```diff
@@ -9,13 +9,13 @@ a git repository and gathering relevant information for a specific purpose.
 ### Project Mission
 **{project mission}** -->
 
-<!-- 
+
 ### Agency Mission
-TODO: Recommended to include since this is an agency-led project -->
+We're a group of civic-minded technologists transforming how the federal government delivers healthcare to the American people.
+
 
-<!-- 
 ### Team Mission
-TODO: Recommended to include since this is an agency-led project -->
+Establish and maintain guidance, policies, practices, and talent pipelines that advance equity, build trust, and amplify impact across CMS, HHS, and Federal Open Source Ecosystems by working and sharing openly.
 
 ## Core Team
```

Comment:
Link in this section is incorrect, please link to github.com. This came from our old README.md template